src/Covid19Dashboard/ChartCarousel/ChartCarousel.less

```diff
@@ -42,3 +42,44 @@
   white-space: pre-wrap;
   z-index: 10; // place it over the "mapbox" link...
 }
+
+/** override some style for carousel's arrow and dots **/
```

Comment:
i'm thinking maybe later we can refactor the carousel component in covid19page into using `antd carousel` as wel